src/iosource/PktSrc.cc

```diff
@@ -385,6 +399,14 @@ void PktSrc::Process()
 			}
 		break;
 		}
+	default:
```

Comment:
This default worries me a bit, could you elaborate on your reasoning for adding this as a fallback for an unknown link-type?